geo/Cargo.toml

```diff
@@ -28,6 +29,7 @@ geographiclib-rs = { version = "0.2.3", default-features = false }
 log = "0.4.11"
 num-traits = "0.2"
 proj = { version = "0.27.0", optional = true }
+geos = { version = "9.1.1", features = ["geo"], optional = true }
```

Comment:
Just to check: we only build `geos` if `geos-tests` is enabled (it isn't by default)?

Also: note that libgeos is LGPL. I'm not sure whether "building bindings using the source" counts as use for licensing purposes, but the source is bundled in the geos crate: https://github.com/georust/geos/tree/master/sys

Reply:
> Just to check: we only build geos if geos-tests is enabled (it isn't by default)?

That's correct.

> Also: note that libgeos is LGPL. I'm not sure whether "building bindings using the source" counts as use for licensing purposes.

Thanks, I somehow never realized geos was LGPL. I'm inclined to remove it since its inclusion is not that critical. Bummer though.